fuzz/src/full_stack.rs

```diff
@@ -333,7 +334,7 @@ impl KeysInterface for KeyProvider {
 		))
 	}
 
-	fn sign_invoice(&self, _invoice_preimage: Vec<u8>) -> Result<RecoverableSignature, ()> {
+	fn sign_invoice(&self, hrp_bytes: &[u8], invoice_data: &[u5]) -> Result<RecoverableSignature, ()> {
```

Comment:
Warnings introduced, need leading underscores here and in `chanmon_consistency`

Reply:
fixed